Application/EdFi.Ods.AdminApi/Infrastructure/Services/ClaimSetEditor/GetResourcesByClaimSetIdQueryV53Service.cs

```diff
@@ -1,242 +1,254 @@
+                if (resourceClaim.ParentResourceClaimId == null)
+                {
+                    var createDefaultStrategy = defaultAuthStrategiesForParents.SingleOrDefault(x =>
+                            x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                            x.Action.ActionName == Action.Create.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(createDefaultStrategy);
-                    var readDefaultStrategy = defaultAuthStrategiesForParents.SingleOrDefault(x =>
-                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                                x.Action.ActionName == Action.Read.Value)?.AuthorizationStrategy;
+                    var readDefaultStrategy = defaultAuthStrategiesForParents.SingleOrDefault(x =>
+                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                                x.Action.ActionName == Action.Read.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(readDefaultStrategy);
-                    var updateDefaultStrategy = defaultAuthStrategiesForParents
-                            .SingleOrDefault(x =>
-                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                                x.Action.ActionName == Action.Update.Value)?.AuthorizationStrategy;
+                    var updateDefaultStrategy = defaultAuthStrategiesForParents
+                            .SingleOrDefault(x =>
+                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                                x.Action.ActionName == Action.Update.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(updateDefaultStrategy);
-                    var deleteDefaultStrategy = defaultAuthStrategiesForParents
-                            .SingleOrDefault(x =>
-                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                                x.Action.ActionName == Action.Delete.Value)?.AuthorizationStrategy;
+                    var deleteDefaultStrategy = defaultAuthStrategiesForParents
+                            .SingleOrDefault(x =>
+                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                                x.Action.ActionName == Action.Delete.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(deleteDefaultStrategy);
+                    var readChangesDefaultStrategy = defaultAuthStrategiesForParents
+        .SingleOrDefault(x =>
+            x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+            x.Action.ActionName == Action.ReadChanges.Value)?.AuthorizationStrategy;
+                    AddStrategyToParentResource(readChangesDefaultStrategy);

```

Comment:
@stephenfuqua 
After the changes for 5.3cqe, array of DefaultAuthStrategies and OverrideAuthStrategies are having 5 elements with null value( with ODS API 5.3). Should I consider conditionally adding the element readChangesDefaultStrategy ?

Reply:
Please refactor as needed.